litmus-portal/frontend/src/containers/layouts/InputField/index.tsx

```diff
@@ -13,16 +13,18 @@ interface InputFieldProps {
   helperText?: string;
   validationError: boolean;
   success?: boolean;
+  disabled?: boolean;
   value: string;
-  required: boolean;
+  required?: boolean;
   iconType?: string | undefined;
-  handleChange: (event: React.ChangeEvent<{ value: string }>) => void;
+  handleChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Make handleChange a required parameter. For Search on Settings use InputBase like its in `BrowseWorkflow/HeaderSection.tsx`. Search does not need validation